### ISSUE TITLE
API Docs has no info about attributes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2547]
|<img alt="redbaron's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| redbaron reported|
|-|-|
|Trac ID|trac#2547|
|Type|enhancement|
|Created|2007-03-30 16:29:04Z|

Example is API docs for t.i.p.DatagramProtocol, there is no info about self.transport attirubute.

<details><summary>Searchable metadata</summary>

```
trac-id__2547 2547
type__enhancement enhancement
reporter__redbaron redbaron
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__website website
keywords__ 
time__1175272144000000 1175272144000000
changetime__1194553202000000 1194553202000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="mwhudson's avatar" src="https://avatars.githubusercontent.com/u/672022?s=50" width="50" height="50">](https://github.com/mwhudson)<a name="note_1"></a>|@mwhudson commented|
|-|-|

To say that "API Docs has no info about attributes" is misleading, as there at least two ways attributes are documented:

zope.interface style: http://twistedmatrix.com/documents/current/api/twisted.web2.iweb.IRequest.html
@ivar style: http://twistedmatrix.com/documents/current/api/twisted.internet.task.LoopingCall.html

Now, to say that "API Docs have little info about attributes" is probably quite accurate... care to submit patches? :-)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun removed owner|
|-|-|

I addressed the missing documentation for DatagramProtocol.transport in datagramprotocol-transport-doc-2547

Fixing every single documentation issue is beyond the scope for one ticket.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set owner to @exarkun|
|-|-|

Note that the link in @type isn't present when creating the documention, maybe that's a bug in pydoctor.

Anyway please merge.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve commented|
|-|-|

Forget the first comment that's was a problem with my script.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [21663]) Merge datagramprotocol-transport-doc-2547

Author: exarkun
Reviewer: therve
Fixes [#2547](https://github.com/twisted/twisted/issues/2547)

Add API documentation to DatagramProtocol for the transport attribute.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
